packages/client/src/queries/getFundingProposal.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from "react-query";
+import { IGetFundingProposalData } from "../types";
+
+export const getFundingProposal = (id?: string) => {
+  const result = useQuery<IGetFundingProposalData, Error>(
```

Comment:
Here this wouldn't work because your using a hook (useQuery) in a function that is neither a hook nor a react component